db/index.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/adidasProducts', {useNewUrlParser: true});
+
+const db = mongoose.connection;
+db.on('error', console.error.bind((console, 'connection error:')));
+db.once('open', () => {
+    console.log(`we're connected!`);
+});
+
+const productSchema = new mongoose.Schema({
+    name: String,
+    photos: String
+});
+
+let Product = mongoose.model('Product', productSchema);
+
+const insertOne = (name, photos) => {
+    var product = new Product({
+        name: name,
+        photos: photos
+    });
+    product.save();
```

Comment:
This can be refactored into one line. Instead of creating a new document and then saving it, you could simply pass an object (representative of your new product) into the 'create' method that comes with your Product method. You might need to pass a callback as the second argument.

Example:` Product.create({name: "cap", photos: <stringified array of urls>}, callback)`